promscale/installation/kubernetes.md

```diff
@@ -73,8 +93,18 @@ can provide the database URI, or specify connection parameters.
     ```bash
     helm repo update
     ```
-1.  Open the `values.yaml` configuration file, download the tobs
-    `values.yaml` from [here][tobs-values-yaml] and locate the `connection`
+1.  Create a database called `tsdb` for Promscale data:
+    ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace <namespace> -l release=<releaseName>,role=master)"
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    CREATE DATABASE tsdb WITH OWNER postgres;
+    ```
+1.  Capture the `postgres` user password:
+    ```bash
+    echo $(kubectl get secret --namespace default tobs-credentials -o jsonpath="{.data.PATRONI_SUPERUSER_PASSWORD}" | base64 --decode)
+    ```
+1.  Download the Promscale
+    `values.yaml` from [here][promscale-values-yaml], open the `values.yaml` configuration file, and locate the `connection`
```

Comment:
@charislam @Rajakavitha1 - whatever we choose to do for linking/wording on the TimescaleDB procedure above, we need to do the same thing here. I personally don't mind what @VineethReddy02 did here (linking to the word 'here'), but it's not consistent with the changes above. Decide on one of the two.

Reply:
Hi @VineethReddy02  and @ryanbooz  Here's what the style guide recommends: 
"Write unique, descriptive link text that makes sense without the surrounding text. Don't use phrases such as this document, this article, or click here."

Reply:
What about something like

1.  [Download the Promscale `values.yaml` file][promscale-values-yaml] and edit  the `connection` section in that file with your TimescaleDB connection details:

Reply:
> What about something like
> 
> 1. [Download the Promscale `values.yaml` file][promscale-values-yaml] and edit  the `connection` section in that file with your TimescaleDB connection details:

Reply:
I made a suggestion. Please feel free to make or ignore the change we can always work on editing after the content is published.

Reply:
Applied your suggestion @Rajakavitha1, I think it's consistent now